### PR TITLE
Dynamic Types and Categories HOPL post

### PR DESCRIPTION
**Pull request by maxsnew**
I used the first paragraph of the historical overview section as the attention-grabber.

### CODE REVIEW

File: blog/_src/posts/2017-05-01-hopl-dyn-cats.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+    Title: Categorical Semantics for Dynamically Typed Programming Languages
+    Date: 2017-05-01T12:25:17
+    Tags: HOPL, category theory, dynamic typing, gradual typing, by Max New
+
+In 1969, Dana Scott wrote a paper in which he said untyped lambda
+calculus had no mathematical meaning, 11 years later he wrote a paper
+that organized many of the different semantics he and others had since
+found using the language of category theory.
+
+<!-- more -->
```
Comment (bennn) [inline blog/_src/posts/2017-05-01-hopl-dyn-cats.md]: I vote to move this to line 4
(none of the other HOPL posts have abstracts)

File: blog/_src/posts/2017-05-01-hopl-dyn-cats.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+This latter paper is really the first deserving of the title
```
Comment (bennn) [inline blog/_src/posts/2017-05-01-hopl-dyn-cats.md]: Remove "latter" and add a link?

Comment (maxsnew) [inline blog/_src/posts/2017-05-01-hopl-dyn-cats.md]: I added links in the previous paragraph instead

File: blog/_src/posts/2017-05-01-hopl-dyn-cats.md (hunk 1 above)
Comment (maxsnew) [inline blog/_src/posts/2017-05-01-hopl-dyn-cats.md]: ok

**Pull request merged**